### ISSUE TITLE
Monaco editor line numbers does not meet color contrast ratio

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52420]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : Win10
**Additional Details:**
Color Contrast,MAS1.4.3
**Repro Steps:**
1. Launch VS Code.
2. Navigate to Help(Alt+H) menu option and select "Interactive Editor Playground" from dropdown.
3. Navigate to editor  where code is written.
4. Open color contrast analyzer and check the ratio.

**Actual:**
Color Contrast ratio is less than 4.5:1  for all the numbers lists  written in the editor.
Same issue observed on other themes also.
**Expected:**

Color Contrast ratio should be greater than 4.5:1.
**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The user will not be able to see lists.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}
Does this issue occur when all extensions are disabled?: Yes
**Attachment For Reference:**
![accessibility a11y_vscode_interactiveeditorplayground_colorcontrast1](https://user-images.githubusercontent.com/39204909/41648797-8c0e540c-7497-11e8-841a-e5002167045a.png)

**Comment by misolori** [comment]
This only fails on our light theme

**Comment by isidorn** [comment]
Also this is not a welcome issue, but a general editor one. Removing label
